src/App.js

```diff
@@ -1,43 +1,24 @@
-import "./App.css";
-import logo from "./logo.png";
+import { Routes, Route } from "react-router-dom";
+import { Cart, Homepage, Login, Products, Signup, Wishlist } from "./pages";
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} alt="mockBee logo" width="180" height="180" />
-        <h1 className="brand-title">
-          Welcome to <span>mockBee!</span>
-        </h1>
-        <p className="brand-description">
-          Get started by editing <code>src/App.js</code>
-        </p>
-        <div className="links">
-          <a
-            href="https://mockbee.netlify.app/"
-            target="_blank"
-            rel="noreferrer"
-          >
-            Explore mockBee
-          </a>
-          <a
-            href="https://mockbee.netlify.app/docs/api/introduction"
-            target="_blank"
-            rel="noreferrer"
-          >
-            API Documentation
-          </a>
-          <a
-            href="https://github.com/neogcamp/mockBee"
-            target="_blank"
-            rel="noreferrer"
-          >
-            Contribute
-          </a>
-        </div>
-      </header>
-    </div>
+    <>
+      <Routes>
+        <Route path="/" element={<Homepage />}></Route>
+        <Route path="/cart" element={<Cart />}></Route>
+        <Route path="/wishlist" element={<Wishlist />}></Route>
+        <Route path="/login" element={<Login />}></Route>
+        <Route path="/signup" element={<Signup />}></Route>
+        <Route path="/products" element={<Products/>}></Route>
+        <Route path="/products/fiction" element={<Products category='Fiction'/>}></Route>
+        <Route path="/products/nonfiction" element={<Products category='Non-Fiction'/>}></Route>
+        <Route path="/products/selfhelp" element={<Products category='Self-Help'/>}></Route>
+        <Route path="/products/boxset" element={<Products category='Box-set'/>}></Route>
+        <Route path="/products/justin" element={<Products category='Just In'/>}></Route>
+        <Route path="/products/popular" element={<Products category='Popular'/>}></Route>
+      </Routes>
```

Comment:
Use separate Router file and then import in App

Reply:
Thank you for the suggestions, Diksha. Will keep these in mind while raising PRs in future.